packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,15 +7,28 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-### Uncategorized
+## [30.0.0]
 
+### Added
+
+- Adds a new field `marketData` to the state of `TokenRatesController` ([#4206](https://github.com/MetaMask/core/pull/4206))
+- Adds a new `RatesController` to manage prices for non-EVM blockchains ([#4242](https://github.com/MetaMask/core/pull/4242))
+
+### Removed
```

Comment:
Ah, it looks the "Removed" section needs to come after the "Changed" section.

Reply:
Right, I misread the output of `yarn changelog:validate`
Fixed in https://github.com/MetaMask/core/pull/4337/commits/2130b49defda524514b9f64b249ce7ec3a7b9d4a